docs/plugins/video-button-response.md

```diff
@@ -1,4 +1,6 @@
-# video-button-response plugin
+# video-button-response
+
+Current version: 9.9.9. [See version history](https://github.com/jspsych/jsPsych/blob/main/packages/plugin-video-button-response/CHANGELOG.md).
```

Comment:
How did this become 9.9.9? :thinking:

Reply:
Oh...
> I also tested this by changing the version number in one of the plugin's package.json file and re-running the gulp tasks.

Reply:
Ugh, sorry!! I could have sworn I changed this back after testing ☹️ I'll fix it.